### ISSUE TITLE
Official OBF Google Account?

### ISSUE THREAD

**Issue by cjfields** [OBF/obf-docs#74]
This was an idea related to BOSC planning and mentioned [on the reviewer document](https://drive.google.com/file/d/1G96dvlUytM0CLReBycynP48XWQpVxe95/view?usp=sharing).  Would having an official OBF Google account be beneficial?  

In relation to the comment this would help in organization and planning (namely a Google Calendar for BOSC, but this would also be useful for the OBF Board meetings).  We also have resources such as the OBF Google Drive folder which primarily reside under my Google Drive account.

**Comment by hlapp** [comment]
It seems you are referring to the [How To Apply For Google For Nonprofits](https://thedigitalnonprofit.com/how-apply-google-nonprofits/) guide? Note that OBF is not a registered non-profit. Our fiscal sponsor, SPI, is. The entity to file this application would have to be SPI, and it can't apply repeatedly for every fiscally sponsored project. So this needs to be coordinated with, agreed to by, and executed by SPI.

**Comment by peterjc** [comment]
If this is specific to wanting public Google Calendar(s), does a standard free Google account not suffice?

For BCC2020 such a calendar and account would probably need to be shared between BOSC and Galaxy (at least temporarily).

**Comment by gedankenstuecke** [comment]
Yes, I think the biggest benefit of a GSuite for OBF would be the possibility of creating Google Meets for hangouts/board meetings, allowing for dial-in connections.

**Comment by cjfields** [comment]
@peterjc that would work with the exception of the occasional need for dial-in teleconf as @gedankenstuecke points out (and we can work around); I think we can also use a non-Google domain.  We could simply set up a standard free account and save the password in our standard location :)

**Comment by peterjc** [comment]
We have a standard free account already for BOSC (used for the You Tube uploads), and a few private Google Groups. Would it be better to have a separate Google account for the OBF and/or BOSC calendars? i.e. Keep the YouTube and Calendar usage separate from a security point of view.

Or, do we try for the full suite option with the paperwork associated with getting that...
